storage/google/cloud/storage/client.py

```diff
@@ -26,10 +26,13 @@
 from google.cloud.storage.bucket import Bucket
 
 
+_marker = object()
```

Comment:
Nit (& only change if you end up modifying something else in this PR): We've called this kind of thing `_SENTINEL` elsewhere.